GliaWidgets/View/Chat/Message/ChoiceCardView.swift

```diff
@@ -1,70 +1,61 @@
 import UIKit
-
 final class ChoiceCardView: OperatorChatMessageView {
     var onOptionTapped: ((ChatChoiceCardOption) -> Void)!
-
     private let viewStyle: ChoiceCardStyle
     private let kLayoutMargins = UIEdgeInsets(top: 12, left: 16, bottom: 16, right: 16)
     private let kImageHeight: CGFloat = 200.0
-
     init(with style: ChoiceCardStyle) {
         viewStyle = style
         super.init(with: style)
     }
-
     @available(*, unavailable)
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
-    override func setup() {
-        contentViews.axis = .vertical
-        contentViews.spacing = 12
-
-        contentViews.layer.cornerRadius = 8
-        contentViews.layer.borderWidth = 1
-        contentViews.layer.borderColor = viewStyle.frameColor.cgColor
-
-        contentViews.isLayoutMarginsRelativeArrangement = true
-        contentViews.layoutMargins = kLayoutMargins
-    }
-
     override func appendContent(_ content: ChatMessageContent, animated: Bool) {
         switch content {
         case .choiceCard(let choiceCard):
-            let contentViews = self.contentViews(for: choiceCard)
-            appendContentViews(contentViews, animated: animated)
+            let contentView = self.contentView(for: choiceCard)
+            appendContentView(contentView, animated: animated)
         default:
             break
         }
     }
-
-    private func contentViews(for choiceCard: ChoiceCard) -> [UIView] {
-        var views = [UIView]()
-
+    private func contentView(for choiceCard: ChoiceCard) -> UIView {
+        let view = UIView()
+
+        view.layer.cornerRadius = 8
+        view.layer.borderWidth = 1
+        view.layer.borderColor = viewStyle.frameColor.cgColor
+        let stackView = UIStackView()
```

Comment:
I'd add a newline between these two lines for readability: `view`-related stuff is finished for now, moving onto `stackView`.